// Copyright 2025 Philipp Stephani
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

edition = "2024";

package rules_elisp;

message Manifest {
  enum Root {
    EXECUTION_ROOT = 0;
    RUNFILES_ROOT = 1;
  }

  Root root = 1;
  repeated string load_path = 2;
  repeated string input_files = 3;
  repeated string output_files = 4;
  repeated string tags = 5;
}
